src/Symfony/Component/Cache/Psr16Cache.php

```
@@ -19,6 +20,12 @@
 use Symfony\Component\Cache\Exception\InvalidArgumentException;
 use Symfony\Component\Cache\Traits\ProxyTrait;
 
+if (class_exists(InstalledVersions::class) && InstalledVersions::isInstalled('psr/simple-cache')) {
+    if (version_compare(InstalledVersions::getVersion('psr/simple-cache'), '3.0.0', '>=')) {
+        throw new \RuntimeException(sprintf('psr/simple-cache "%s" is not compatible with this version of symfony/cache. Please upgrade symfony/cache to 6.0+ or downgrade psr/simple-cache to v1.x or 2.x.', InstalledVersions::getPrettyVersion('psr/simple-cache')));
```

Please throw a Logo exception instead

---

I've force-pushed 9a729120ba which throws a `LogicException` instead